### PR TITLE
333 podcast

### PR DESCRIPTION
**Pull request by theabrad**

### CODE REVIEW

File: front-end/src/components/LearningEvent/LearningEventPodcast.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  state = {
+    url: this.props.url,
+    playing: false,
+    volume: 0.8,
+    muted: false,
+    pip: false,
+    played: 0,
+    loaded: 0,
+    duration: 0,
+    playbbackRate: 1.0,
```
Comment (kdyer) [inline front-end/src/components/LearningEvent/LearningEventPodcast.js]: typo?

File: front-end/src/components/LearningEvent/LearningEventPodcast.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    loop: false,
+    height: '100%',
+    width: '100%'
+  }
+
+  render() {
+    return (
+      <div>
+        This is a podcast.
```
Comment (kdyer) [inline front-end/src/components/LearningEvent/LearningEventPodcast.js]: Is this the content?

Comment (theabrad) [reply front-end/src/components/LearningEvent/LearningEventPodcast.js]: Ticket 331 is adding the player controls. This was just to make sure the component loads.

Comment (theabrad) [reply front-end/src/components/LearningEvent/LearningEventPodcast.js]: I should add the ReactPlayer component in here though. I had it in another branch.

**Pull request merged**